rubocop.yml

```diff
@@ -714,9 +714,7 @@ Style/WordArray:
 ##################### Metrics ##################################
 
 Metrics/AbcSize:
-  # The ABC size is a calculated magnitude, so this number can be a Fixnum or
-  # a Float.
-  Max: 15
+  Enabled: false
```

Comment:
I understand that this is sometimes ignored. I do that myself when there is a good reason. But when we disable it here we will have to keep an extra eye on long methods and classes in the QA reviews. So no hard limit but discussions during QA. I'm strongly against long methods and classes.
Or another suggestion would be to work with Rubocop's "ignore comments": https://medium.freecodecamp.org/rubocop-enable-disable-and-configure-linter-checks-for-your-ruby-code-475fbf11046a